### PR TITLE
Update script to incorporate touch

### PR DESCRIPTION
**Pull request by ShoGinn**
Just a simple change to incorporate touch

### CODE REVIEW

File: snapraid-runner.py [hunk 1]
```diff
@@ -241,6 +243,14 @@ def run():
                       config["snapraid"]["config"])
         finish(False)
 
+    if config["snapraid"]["touch"]:
+	logging.info("Running touch...")
+	try:
+		snapraid_command("touch")
+	except subprocess.CalledProcessError as e:
+		logging.error(e)
+		finish(False)
```
Comment (Chronial) [inline snapraid-runner.py]: This exception handling is not needed – it is already done in `main`.

File: snapraid-runner.py [hunk 2]
```diff
@@ -241,6 +243,14 @@ def run():
...
+	logging.info("*" * 60)
```
Comment (Chronial) [inline snapraid-runner.py]: Please add an empty line below this, so each step has its own block.

File: snapraid-runner.conf.example [hunk 3]
```diff
@@ -5,6 +5,8 @@ executable = snapraid
 config = snapraid.conf
 ; abort operation if there are more deletes than this, set to -1 to disable
 deletethreshold = 40
+; if you want touch to be ran each time
+touch = true
```
Comment (Chronial) [inline snapraid-runner.conf.example]: I am not convinced that this should be on by default.

File: snapraid-runner.py [hunk 4]
```diff
@@ -152,6 +152,8 @@ def load_config(args):
     config["scrub"]["enabled"] = (config["scrub"]["enabled"].lower() == "true")
     config["email"]["short"] = (config["email"]["short"].lower() == "true")
 
+    config["snapraid"]["touch"] = (config["snapraid"]["touch"].lower() == "true")
```
Comment (Chronial) [inline snapraid-runner.py]: Why is there an empty line before this line?

File: snapraid-runner.conf.example (hunk 3 above)
Comment (ShoGinn) [inline snapraid-runner.conf.example]: I agree, keep it as an option and not make it mandatory in the config

File: snapraid-runner.py (hunk 4 above)
Comment (ShoGinn) [inline snapraid-runner.py]: No reason, just bad editing.

**Comment by Chronial** [comment]
Looks great, thanks :)

**Pull request merged**